Description: Fix storage.prefix does not exist in Django 1.7
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-09-30

--- python-django-pyscss-1.0.3.orig/django_pyscss/utils.py
+++ python-django-pyscss-1.0.3/django_pyscss/utils.py
@@ -1,5 +1,6 @@
 import fnmatch
 import os
+import collections
 
 from django.contrib.staticfiles import finders
 
@@ -11,8 +12,16 @@ def find_all_files(glob):
     storage objects must implement the File storage API:
     https://docs.djangoproject.com/en/dev/ref/files/storage/
     """
+    found_files = collections.OrderedDict()
     for finder in finders.get_finders():
         for path, storage in finder.list([]):
-            if fnmatch.fnmatchcase(os.path.join(storage.prefix or '', path),
-                                   glob):
+            if getattr(storage, 'prefix', None):
+                prefixed_path = os.path.join(storage.prefix, path)
+            else:
+                prefixed_path = path
+            
+            if prefixed_path not in found_files:
+                found_files[prefixed_path] = (storage, path)
+
+            if fnmatch.fnmatchcase(prefixed_path, glob):
                 yield path, storage
